Local build is broken

### Description

Local build is broken when running `.\build.ps1`j:
Reason for this is, that `Microsoft.NET.Test.Sdk` must not be included 

### Reproduction Steps

```
.\build.ps1
```


### Expected behavior

The build should succeed

### Actual behavior

The build fails with
```
╬══════════════════════
║ Errors & Warnings
╬═════════════
​
[ERR] Compile: C:\temp\AwesomeAssertions\Tests\Benchmarks\Program.cs(13,24): error CS0017: Program has more than one entry point defined. Compile with /main to specify the type that contains the entry point. [C:\temp\AwesomeAssertions\Tests\Benchmarks\Benchmarks.csproj::TargetFramework=net8.0]
[ERR] Compile: C:\temp\AwesomeAssertions\Tests\Benchmarks\Program.cs(13,24): error CS0017: Program has more than one entry point defined. Compile with /main to specify the type that contains the entry point. [C:\temp\AwesomeAssertions\Tests\Benchmarks\Benchmarks.csproj::TargetFramework=net8.0]
[ERR] Compile: Target Compile has thrown an exception
​
═════════════════════════════════════════════════
Target                       Status      Duration
─────────────────────────────────────────────────
InstallNode                  Succeeded       0:01   // Skipped: Downloading and extracting
SpellCheck                   Succeeded       0:03
Clean                        Succeeded     < 1sec
Restore                      Succeeded       0:03
CalculateNugetVersion        Succeeded     < 1sec
Compile                      Failed          1:30   // Version: 9.3.1-foo.1
```

### Regression?

_No response_

### Known Workarounds

_No response_

### Configuration

_No response_

### Other information

This broke with #364 when adding `Microsoft.NET.Test.Sdk` to the `Tests/Directory.Build.props`, and with this also the the benchmark project.

### Are you willing to help with a pull-request?

Yes, please assign this issue to me.

--- a/Tests/Directory.Build.props
+++ b/Tests/Directory.Build.props
@@ -2,7 +2,7 @@
   <!-- inherit solution wide configuration -->
   <Import Project="../Directory.Build.props"/>
 
-  <ItemGroup>
+  <ItemGroup Condition="$(MSBuildProjectName.EndsWith('.Specs')) OR $(MSBuildProjectName.EndsWith('.Tests'))">
     <!--
     Starting version 17.14.0 net6 support has been removed.
     As long as we want to build for net6 we need to reference version 17.13.

